snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -1,14 +1,30 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  labels:
+    app.kubernetes.io/instance: snyk-monitor
+    app.kubernetes.io/version: 0.0.0
+    app.kubernetes.io/name: snyk-monitor
+    app.kubernetes.io/managed-by: operator
   annotations:
     alm-examples: |-
       [
         {
           "apiVersion": "charts.helm.k8s.io/v1alpha1",
           "kind": "SnykMonitor",
           "metadata": {
-            "name": "snyk-monitor"
+            "name": "snyk-monitor",
+            "labels": {
+              "app.kubernetes.io/instance": "snyk-monitor",
+              "app.kubernetes.io/version": "0.0.0",
+              "app.kubernetes.io/name": "snyk-monitor",
+              "app.kubernetes.io/managed-by": "operator"
+            },
+            "annotations": {
+              "productID": "OVERRIDE_TO_METERING_PRODUCT_ID",
+              "productName": "snyk-monitor",
+              "productVersion": "0.0.0"
+            }
```

Comment:
Was this auto-generated? I think it would come from the CR/CRD, or does it get auto-populated in the UI using these values?